src/backend/core/migrations/0011_remove_resource_is_public_room_access_level.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 5.1.5 on 2025-02-16 11:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0010_alter_resourceaccess_options_alter_user_options'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='resource',
+            name='is_public',
+        ),
```

Comment:
This one is not used yet, right? I just wonder if you need to migrate data or not.